config.json.example

```diff
@@ -7,6 +7,8 @@
     "FACEBOOK_ANALYTICS":"1234567890123456",
     "FACEBOOK_KEY":"123456789012345",
     "FACEBOOK_SECRET":"aaaabbbbccccddddeeeeffff00001111",
+    "GOOGLE_CLIENT_ID":"123456789012345",
```

Comment:
Not something we have to do right away (so it doesn't block this PR), but we should add a description to the wiki for how to create creds for a local install. Same with Facebook.